### DISCUSSION TITLE
CSS selector for grid column

### DISCUSSION THREAD

**Discussion by davetapley** [Textualize/textual#2146] [Q&A] [answered]
Is there a CSS selector for a specific grid column?

In my use case I have two columns of `Static` and I want to set `text-align: right` on the first column, e.g.:

```
      foo | boo
   longer | pip
```

**Comment by TomJGooding** [comment, marked as answer]
> Is there a CSS selector for a specific grid column?

I'm not sure that could work with all the possible layouts using `Grid`?  Here's the demo of [All padding settings](https://textual.textualize.io/styles/padding/#all-padding-settings) from the docs, I would follow that example and uses classes/id's to specify formatting.
